docs/podman-containers-cleanup.1.md

```diff
@@ -0,0 +1,38 @@
+% podman-container-cleanup "1"
+
+## NAME
+podman\-container\-cleanup - Cleanup Container storage and networks
+
+## SYNOPSIS
+**podman** **container-cleanup** [*options* [...]] container
+
+## DESCRIPTION
+`podman container cleanup` will cleanup one or more containers from the host.  The container name or ID can be used.  This does not remove containers.  Running containers will not be cleaned up.
```

Comment:
suggest "remove containers" to "remove the container"

Reply:
oops, forgot the --all flag..  suggest adding 'the' only then.  "remove the containers".